### ISSUE TITLE
tf.depth_to_space / tf.space_to_depth

### ISSUE THREAD

**Issue by hardmaru** [Tetrachrome/subpixel#4]
Have you looked at implementing this using built-in functions in TensorFlow?  Should be much more efficient.

**Comment by EderSantana** [comment]
Yeah I just heard about good news about depth_to_space https://twitter.com/poolio/status/782819206517497856

It will make the code cleaner. Thanks for letting me know about it
